javascript/node/selenium-webdriver/chromium.js

```diff
@@ -870,6 +871,36 @@ class Driver extends webdriver.WebDriver {
       }
     })
   }
+
+  /**
+   * @param connection
+   * @param callback
+   * @returns {Promise<void>}
+   */
+  async logMutationEvents(connection, callback) {
+    await connection.execute('Runtime.enable', 1, {}, null)
+    await connection.execute('Page.enable', 2, {}, null)
+
+    await connection.execute('Runtime.addBinding', 3, {
+      name: '__webdriver_attribute',
+    }, null)
+
+    const mutationListener = fs.readFileSync('../../cdp-support/mutation-listener.js', 'utf-8').toString()
+
+    await this.executeScript(mutationListener)
+
+    await connection.execute('Page.addScriptToEvaluateOnNewDocument', 4, {
```

Comment:
what does the `4` in this line mean. I am guessing it's some form of execution context but can we change this to not be a magic number.

sorry if you have done it this way before but it would be better to have something else for this (even if its `context++`

Reply:
Its a general id that cdp needs, nothing useful as such to us, but yeah, can make it like `context++`